src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -66,8 +73,6 @@ public ExceptionLayoutRenderer()
             this.MaxInnerExceptionLevel = 0;
         }
 
-        private delegate void ExceptionDataTarget(StringBuilder sb, Exception ex);
-
         /// <summary>
         /// Gets or sets the format of the output. Must be a comma-separated list of exception
```

Comment:
please refer to Formats + ExceptionRenderingFormat

with
`<see cref="Formats"/>` and `<see cref="ExceptionRenderingFormat"/>`